cartoframes/layer.py

```diff
@@ -184,15 +184,10 @@ class QueryLayer(AbstractLayer):
             values described:
 
             - column (str): Column to base coloring from.
-            - scheme (str, optinal): Color scheme from
-              `CartoColors
+            - scheme (dict, optional): Dictionary definition of color scheme from
+              styling module of cartoframes that exposes `CartoColors
               <https://github.com/CartoDB/CartoColor/wiki/CARTOColor-Scheme-Names>`__.
-              Defaults to `Mint`.
-            - bin_method (str, optional): Quantification method for dividing
-              data range into bins. Must be one of: ``quantiles``, ``equal``,
-              ``headtails``, or ``jenks``. Defaults to ``quantiles``.
-            - bins (int, optional): Number of bins to divide data amongst in
-              the `bin_method`. Defaults to 5.
+              Defaults to mint scheme.
```

Comment:
Could you also add something to the effect of "..., or a scheme from the styling module such as `styling.sunset(7)`"?

Reply:
done!